### PR TITLE
Milestone: 8 Add pop-up for mobile and desktop version

### PR DESCRIPTION
**Pull request by muhammedb8826**
## I have created db.js and stored data of the project to object:
- description, 
- featured image,
-  technologies,
-  link to live version,
-  link to the source. 
## I have created work.js to implement the modal for mobile and as well as for desktop

### CODE REVIEW

File: scripts/works.js [hunk 1]
```diff
@@ -135,21 +144,180 @@ btn0.addEventListener('click', () => {
...
+  li33.className = 'title-and-year';
+  li33.textContent = `${data[3].technologies[0].teck3}`;
+  div2.className = 'live-and-source-btn';
+  seeLive.className = 'btns';
+  seeSource.className = 'btns';
+  seeLive.textContent = 'See Live';
+  seeSource.textContent = 'See Source';
+  liveIcon.setAttribute('src', '../assets/live-icon.png');
+  liveIcon.className = 'live-icon';
+  seeLive.append(liveIcon);
```
Comment (GedeonTS) [inline scripts/works.js]: - [ ] :thinking: I suggest using avoiding to repeat yourself implementing the onlick event at each 'see-project' in order to stick to meet the javascript best practices [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=simplicity.-,DRY.,-If%20you%20use) 

HINT: for that you can create a loop for these buttons and take the from the project array data by the button index

File: style.css [hunk 2]
```diff
@@ -150,17 +143,167 @@ nav.active .menu-items li {
...
+  text-decoration: none;
+  border-radius: 16px;
+  color: #396df2;
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  cursor: pointer;
+  transition: background-color 1s ease;
+}
```
Comment (GedeonTS) [inline style.css]: - [] Kindly make sure to stick to the design by letting all your popup display and scrolling the whole overlay instead :recycle: 

HINT: for this you can set overflow hidden for the popup, after making sure it's heignt is auto, then set overflow-y:scroll for the overlay section :recycle: 

Your design:

Screenshot from 2023-02-04 21-20-15

Figma design:

Screenshot from 2023-02-04 21-30-14

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @muhammedb8826 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- No linter errors :+1: 
- The popup interactions are working :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by huseyinbiyik — APPROVED** [review]
Hi @muhammedb8826,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- PR description is descriptive 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions
⚠️ Unfortunately, the previous reviewer didn't notice that you worked solo. Normally, this branch of the project needs to be completed with your pair programming partner. Kindly check the [Work and Submission Mode](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#work-and-submission-mode) for the next time 👍

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._∂

**Pull request merged**